src/content/docs/swate/term-search.mdx

```diff
@@ -68,6 +68,18 @@ To trigger this search the input must match the [Swate Regex pattern](http://reg
 To execute the search query on any search text again after closing the dropdown with the search results, double click into the search field.
 :::
 
+## Adapting the Term Search
+
+In the settings you can configure your term search in the "Term Search Configuration" part further. You can decide whether the default term search shall be used or not. Additionally, you can add and remove high performance search TIB terms to optimize your search further.
+
+##### Default Search
+
+By using the default search, users also query the [DPBO](https://nfdi4plants.github.io/nfdi4plants.knowledgebase/fundamentals/ontology-service-landscape/#how-can-i-find-a-certain-ontology-term-for-my-data-annotation), an ontology repository that is quickly updated based on user feedback. That way new terms are often added faster here than in other ontologies and enable the usage of these new term.
```

Comment:
```suggestion
Default search enables/disables search of the [SwateDB](/fundamentals/ontology-service-landscape/#swatedb-and-swobup). A DataPLANT maintained ontology database, which features a set of external ontologies, as well as our DPBO broker ontology, allowing us to quickly update and extend the terms offered via this search option.
```

Remember to verify correct link. And do not use absolute links to live service, instead use relative links.